### PR TITLE
(DOCSP-25415): networkTraffic config option

### PR DESCRIPTION
**Pull request by jeff-allen-mongo**
## DESCRIPTION

The top-level configuration file page is currently progress (being worked on by @jocelyn-mendez1 ). For now, I'm basing these changes off of her branch and creating a sub-page for this specific option. The top-level config file page will be more fleshed out prior to publish.

Note that I'm not totally sure about the best approach for these options in general. I think we should definitely have some kind of reference page that lists all available options, but having specific pages showing how to use the options to specific tasks (like the page created in this PR) might also be useful.

## JIRA

https://jira.mongodb.org/browse/DOCSP-25415

## STAGING

https://docs-mongodbcom-staging.corp.mongodb.com/compass/docsworker-xlarge/DOCSP-25415/config-file/restrict-outgoing-connections/

## BUILD LOG

https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=637505d17f102f5ceaba0d84

### CODE REVIEW

File: source/config-file/restrict-outgoing-connections.txt [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+.. note::
+
+  The name and filepath of the |compass-short| executable depend on your
+  operating system.
+
+Configuration File Example
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can specify the |compass-short| configuration file in either EJSON or
```
Comment (jocelyn-mendez1) [inline source/config-file/restrict-outgoing-connections.txt]: [Suggestion]

Maybe for consistency add something like  "The following command sets the --no-networkTraffic option using the configuration file" or similar to the sentence in Command Line Example ?

**Review by jocelyn-mendez1 — APPROVED** [review]
Hi @jeff-allen-mongo, this LGTM! I just left one minor suggestion. Thanks!

File: source/config-file/restrict-outgoing-connections.txt (hunk 1 above)
Comment (jeff-allen-mongo) [reply source/config-file/restrict-outgoing-connections.txt]: Good idea, done.

File: source/config-file/restrict-outgoing-connections.txt [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+You can configure |compass| to not perform outgoing network operations
+other than those to the database. This lets you configure any
+:ref:`edition <compass-feature-table>` of |compass-short| to restrict
+outgoing connections in the same way :ref:`Compass Isolated Edition
+<compass-faq-isolated>` restricts connections.
+
+Consideration
+-------------
+
+When you block outgoing network connections, |compass-short| can't use
```
Comment (addaleax) [inline source/config-file/restrict-outgoing-connections.txt:24]: (I know we’re not to look for stylistic issues, so feel free to disregard, but I think our docs don’t typically use contractions)

**Review by addaleax — APPROVED** [review]
looks great, thank you!

**Pull request merged**